src/targeting.js

```diff
@@ -443,7 +443,7 @@ export function newTargeting(auctionManager) {
   }
 
   function getCustomKeys() {
-    let standardKeys = getStandardKeys();
+    let standardKeys = getStandardKeys().concat(NATIVE_TARGETING_KEYS);
```

Comment:
If `NATIVE_TARGETING_KEYS` are part of standard keys, shouldn't we move this into the `getStandardKeys()` function?

Reply:
If we moved it there, then the native keys would be affected by the `bidderSettings.sendStandardTargeting` property (which currently only focuses on the base keys via this logic [here](https://github.com/prebid/Prebid.js/blob/master/src/targeting.js#L384)).

At the moment, the native keys are still written when that flag is set to `false`.  If it was moved, then the natives aren't written.  Is that the behavior we want?

Reply:
I suppose that would be a breaking change, so we leave as is for now. Thanks